spaces/S000147/properties/P000097.md

```diff
@@ -1,7 +1,7 @@
 ---
 space: S000147
-property: P000057
-value: false
+property: P000097
```

Comment:
We could add https://topology.pi-base.org/properties/P000065 and https://topology.pi-base.org/properties/P000114 (the space assumes existence of CH).

Reply:
I guess so, and also for S148 (the other Luzin space).  I'll add it.

Reply:
Question: For spaces whole definition relies on extra set-theoretic assumptions, how about adding them in parentheses at the beginning of the space name?  So in this case: `(CH) A Luzin subset of the reals (Scheepers 1999)` for example.

Reply:
I like that